packages/commons-test/src/containerTestUtils.ts

```diff
@@ -15,6 +15,11 @@ export const TEST_MINIO_PORT = 9000;
 export const TEST_MINIO_IMAGE =
   "quay.io/minio/minio:RELEASE.2024-02-06T21-36-22Z";
 
+export const TEST_MAILPIT_HTTP_PORT = 8025;
+export const TEST_MAILPIT_SMTP_PORT = 1025;
+export const TEST_MAILPIT_IMAGE = "axllent/mailpit";
+export const TEST_EMAIL_MANAGER_SENDER = "sender@test.com";
```

Comment:
This `TEST_EMAIL_MANAGER_SENDER ` seems to be not needed for the container setup